### DISCUSSION TITLE
nonebot-plugin-test 插件停止维护？

### DISCUSSION THREAD

**Discussion by deltazefiro** [nonebot/discussions#179] [问答] [answered]
之前一直在使用 [nonebot-plugin-test](https://github.com/nonebot/plugin-test)，系统重装后找这个插件找了好久发现已经停止维护了  
 且 `plugin-test` 在 `pip` 发布的最后的版本于目前 `Nonebot 2.0.0-beta.4` 上无法正常运行。
感觉这个插件相当实用 它停止维护的原因是？有其他替代的插件吗？  
如果没有，那么 `plugin-test` 是否存在严重问题？能否简单地修复？  
谢谢。

**Comment by yanyongyu** [comment, marked as answer]
目前单元测试可以使用pytest+nonebug进行，之后可能可以使用console适配器（WIP）来进行交互式测试

停止维护的主要原因是插件过于依赖cqhttp而难以持续更新。

**Reply by deltazefiro** [reply]
理解了  
`nonebug` 确实在做系统化测试上更好  
但毕竟交互式还是更方便直接 期待交互式调试器的开发  
谢谢
